approved.yaml

```diff
@@ -70,16 +70,24 @@ thestrange:
 
 #Publisher Sheets
 #When adding an official sheet include 'official' in the short name for the select menu
-earthdawnofficial:
-  - "Earthdawn (FASA Official)"
-  - "Earthdawn by FASA"
-  - "Earthdawn"
-
 bandofbladesofficial:
   - "Band of Blades Official"
   - "Band of Blades by Evil Hat Productions"
-  - "Blades in the Dark"
+  - "Forged in the Dark"
 
+bladesinthedarkadvancedofficial:
+- "Blades in the Dark"
```

Comment:
@clevett what's the reason for this change? FYI, this has broken every single game for Blades (and S&V too, I presume) by forcing users to reselect sheet in game settings (and it's not obvious they need to do that).

Reply:
@joesinghaus 

*official* is required in the short name to set the **sheet by Publisher**.  An oversight on my part as you're right this results in the sheet no longer being selected. I'll need to make an adjustment in the future to better handle these updates.

Reply:
@clevett Right, no worries, there were just a lot of not-so-tech-savvy people confused as to where their character sheet went. Maybe in the future it would be a good idea to migrate the selected sheet for all existing users along with such a change.